go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -149,8 +149,8 @@ func init() {
 	flag.BoolVar(&currentConfig.CacheResultFields, "enable-query-plan-field-caching", defaultConfig.CacheResultFields, "This option fetches & caches fields (columns) when storing query plans")
 
 	flag.DurationVar(&healthCheckInterval, "health_check_interval", 20*time.Second, "Interval between health checks")
-	flag.DurationVar(&degradedThreshold, "degraded_threshold", 30*time.Second, "replication lag after which a replica is considered degraded (only used in status UI)")
```

Comment:
Is this really used anywhere other than Status UI?

Reply:
We populate the healthStreamer with this info, so potentially, yes?

Reply:
oh yes, that is new. This is fine.